trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class TrimmerDecorator < BaseDecorator
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name[0...10]
+  end
+end
```

Comment:
- [ ] Please require base_decorator here also because  TrimmerDecorator is making use of it.  If the file is not required, you will encounter an `uninitialized constant BaseDecorator` error when testing.

```ruby
require './base_decorator.rb'

class TrimmerDecorator < BaseDecorator
   # content goes here
end
```

Reply:
🟢